Scripts/LevelManager.gd

```diff
@@ -53,10 +53,7 @@ func on_Dwarf_died():
 	killed_dwarves += 1
 	emit_signal("dwarf_died")
 	
-	if tavern_enter_btn.pressed:
-		dwarves_spawner.spawn_tavern()
-	else:
-		spawn_next_dwarf()
+	spawn_next_dwarf()
```

Comment:
I missunderstand why You removed this if-else statement? How to enter tavern in this moment? EnterTavernBtn is not working.

Reply:
Oh, mistake. I was wrong with another similar place